Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN locale-gen fr_FR \
-    && locale-gen en_US.UTF-8 \
-    && dpkg-reconfigure locales \
-    && update-locale LANG=en_US.UTF-8 \
-    && update-locale LC_ALL=en_US.UTF-8 \
-    && ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
-    && locale-gen \
-    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
-
-ENV PYTHONIOENCODING="utf-8" \
-    LANG="C.UTF-8" \
```

Comment:
Setting the variables LANG, LC_ALL and LANGUAGE and then rebuilding the locales is enough to configure a sane locale environment avoiding the use of PYTHONIOENCODING, which is [considered bad practice](http://stackoverflow.com/a/34378962).

Here you can see the output without configuration:
```
root@3e24b6cc2488:/# python
Python 2.7.11+ (default, Mar 30 2016, 21:00:42) 
[GCC 5.3.1 20160330] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.stdout.encoding
'ANSI_X3.4-1968'
>>> print u"\u20AC"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\u20ac' in position 0: ordinal not in range(128)
```

And after configuration [as implemented in the script](#diff-d2ff34f73419a2f54edcc6a5fc1f020aR60):
```
root@3e24b6cc2488:/# python
Python 2.7.11+ (default, Mar 30 2016, 21:00:42) 
[GCC 5.3.1 20160330] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.stdout.encoding
'UTF-8'
>>> print u"\u20AC"
€
```